project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 //addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.8.4")
 
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.0")
+//addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.1.0")
```

Comment:
I've re-instated sbt-updates 0.3.0 per the custom resolver given at https://github.com/rtimush/sbt-updates.